test/steps/mappers/shared_functions.go

```diff
@@ -20,20 +20,23 @@ import (
 	"github.com/cucumber/messages-go/v10"
 )
 
+// TableRow represents a row of godog.Table made to a step definition
+type TableRow = messages.PickleStepArgument_PickleTable_PickleTableRow
```

Comment:
There is already this aliasing defined in the `steps` package. Can't we use that?

Reply:
The alias was located in ` test/steps/utils.go ` which is deleted, so moved the alias here.

Reply:
Ah, I see. Thanks.